tasks/db_deploy/test/manual_tests/README.md

```diff
@@ -7,7 +7,6 @@ To run the tests provided below, run the steps in the [initial setup](#initial-s
 section first.
 
 It is recommended to run the tests in the following order.
-- [Database Does Not exist Failure Test](#database-does-not-exist-failure-test)
```

Comment:
I think this  is still valid, the only difference should be the output. Instead of an error, the database should be created and populated. Maybe combine some of the setup with the fresh install?

Reply:
Moved detail to the fresh install test